articles/virtual-machines/metadata-security-protocol/other-examples/audit-logs-to-rules.md

```diff
@@ -38,8 +38,7 @@ An allow list is comprised of privileges, identities, roles, and role assignment
 ### Automated Allowlist generation
 
 1. Download and run the Allowlist Tool from either option:
```

Comment:
```suggestion
1. Download and run the Allowlist Tool:
```
Deleting the text about "either option" now that there's only one option in the list @minnielahoti